transport/tls/stream_dialer_test.go

```diff
@@ -44,10 +44,10 @@ func TestUntrustedRoot(t *testing.T) {
 	require.ErrorAs(t, err, &certErr)
 }
 
-func TestRevoked(t *testing.T) {
+func TestExpired(t *testing.T) {
 	sd, err := NewStreamDialer(&transport.TCPDialer{})
 	require.NoError(t, err)
-	_, err = sd.DialStream(context.Background(), "revoked.badssl.com:443")
```

Comment:
I explicitly wanted to test for revoked here. I don't think the issue is with the original test. It seems like `revoked.badssl.com` no longer serves an expired cert, so we don't return an error. Chrome is considering it safe.
<img width="369" alt="image" src="https://github.com/Jigsaw-Code/outline-sdk/assets/113565/b3785138-01cc-44fc-94bc-517cebfc98e0">

Can you instead comment it out or use `t.Skip("Skipping until revoked.badssl.com is fixed")` in addition to adding the expired test?